src/Cli/dotnet/commands/dotnet-workload/install/NetSdkManagedInstaller.cs

```diff
@@ -190,7 +196,9 @@ public void InstallWorkloadManifest(ManifestId manifestId, ManifestVersion manif
             string packagePath = null;
             string tempExtractionDir = null;
             string tempBackupDir = null;
-            var manifestPath = Path.Combine(_dotnetDir, "sdk-manifests", sdkFeatureBand.ToString(), manifestId.ToString());
+            string rootInstallDir = WorkloadInstall.IsUserLocal(_dotnetDir, sdkFeatureBand.ToString()) ? _userProfileDir : _dotnetDir;
+            // TODO: do we need to use something from _dotnetDir sdk-manifests for user local install?
```

Comment:
Is there a need to use the workload folder from the dotnet root here when we do a user local install?